pandas/tests/indexes/test_base.py

```diff
@@ -1964,6 +1967,9 @@ def test_isin_nan_common_float64(self, nulls_fixture):
             pytest.skip("pd.NaT not compatible with Float64Index")
 
         # Float64Index overrides isin, so must be checked separately
+        if nulls_fixture is pd.NA:
+            pytest.xfail("Not sure how to handle with Float64Index?")
```

Comment:
Not sure about this test. Maybe logical to support, though we also skip for `pd.NaT` atm

Reply:
I don't think we should support converting pd.NA into np.nan in creating a float index, since pd.NA and np.nan behave differently